lib/robot.js

```diff
@@ -241,4 +241,9 @@ module.exports = (...args) => new Robot(...args);
   * robot.log.warn("Woah there");
   * robot.log.error("ETOOMANYLOGS");
   * robot.log.fatal("Goodbye, cruel world!");
+  *
+  * Note: Probot supports execption tracking through raven, a client for
+  * [sentry](https://github.com/getsentry/sentry). If the `SENTRY_DSN` is set
+  * as an environment varialbe, all errors will be forwarded to the sentry host
+  * specified by the environment variable.
```

Comment:
I believe it forwards everything: https://github.com/transcovo/bunyan-sentry-stream/blob/1485adfff3df22db2e86f08a70fc2baddd336c02/lib/SentryStream.js#L33-L36